### DISCUSSION TITLE
How to permit Nuxt 3 API authentication in Laravel?

### DISCUSSION THREAD

**Discussion by Arturexe** [nuxt/nuxt#16186] [Questions]
I'm trying to send a `post request` from Nuxt 3 frontend to an `auth api route` in my Laravel backend but it results in following error:

    Access to fetch at 'http://127.0.0.1:8000/api/authenticate' from origin 'http://localhost:3000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

This is the function, that creates the request:

    const submit = () => {
        return $fetch('http://127.0.0.1:8000/api/authenticate', {
            method: 'POST',
            body: form //data proxy (const form = reactive({...})
        })
    }

api route:

    Route::post('/authenticate', [AuthenticatedSessionController::class, 'test']);

controller:

    public function test(Request $request)
    {
        dd($request);
    }

This [stack question][1] tells to put this into `nuxt.config.js`:

    axios: {
        proxy: true
      },
    proxy: {
        "/api": "https://127.0.0.1:8000"
      },

and use `/api` endpoint in `fetch` function but it doesn't change anything.

Nuxt runs on `localhost:3000` and Laravel on `localhost:8000`

  [1]: https://stackoverflow.com/questions/70755026/in-nuxt-js-getting-cors-problem-no-access-control-allow-origin-header-is-prese

**Comment by wobsoriano** [comment]
Enable cors on your Laravel app? https://github.com/fruitcake/laravel-cors

**Comment by IsraelOrtuno** [comment]
You can find more in Laravel documentation website:
https://laravel.com/docs/9.x/routing#cors

There's a `core.php` file where you can easily configure what you need.

**Reply by Arturexe** [reply]
How exactly would I configure `cors.php` for laravel to accept the nuxt 3 `proxy request`?
